cli/js/deno.ts

```diff
@@ -118,3 +118,4 @@ export let pid: number;
 export let noColor: boolean;
 
 export { symbols } from "./symbols.ts";
+export { testFunction } from "crate://deno_core/test_file.ts";
```

Comment:
This is pretty cool.

Reply:
@afinch7 can you convert it to `import` so it's not exposed?

Reply:
Seems to work fine as well, but I will remove it now that I know that it works.
```
./target/debug/deno                                                                                                            ░▒▓ ✔  1m 24s 
> Deno.testFunction("test");
Hello from testFunction: test
```